helper/config/custom_types.go

```diff
@@ -0,0 +1,72 @@
+package config
+
+import (
+	"strconv"
+)
+
+type Trilean uint8
```

Comment:
Super nice I think this is the right place and wouldn't have thought of that !! 👍 👍 👍 👍 👍

Reply:
Also, after some tweaking around, I realised that it was possible to do something like this: 
```suggestion
type Trilean *bool
```